tests/attestation_plugin/plugin/tests.c

```diff
@@ -308,6 +304,62 @@ static void _test_verify_evidence_fail()
     OE_TEST(oe_free_endorsements(endorsements) == OE_OK);
 }
 
+static void _test_custom_claims_serde()
```

Comment:
What's `serde`? Please spell out.

Reply:
renamed the test to _test_custom_claims_seriaize_deserialize()